NAMESPACE

```diff
@@ -140,10 +136,7 @@ import(maftools)
 import(metaviz)
 import(parallel)
 import(plotly)
-import(randomForest)
```

Comment:
I guess this fix solves issue #160. If that's the case, I can revert the fix I added to my branch (i.e [this](https://github.com/morinlab/GAMBLR/pull/164/files#diff-2ee57637b77c38eb625492f80e8c6d88b08103cce85eb89198a2e7f2f3b1832fR2154) and [this](https://github.com/morinlab/GAMBLR/pull/164/files#diff-2ee57637b77c38eb625492f80e8c6d88b08103cce85eb89198a2e7f2f3b1832fR2396))

Reply:
Please keep your fixes. It is a good practice to refer to the function through the package. I did not mean to "override" your fixes - they will still be needed to ensure reproducibility.

Reply:
For sure!